File: Eclipse 2/codeLM starter-code/Python/poker_table_traversal.py
def poker_table_traversal(tables):
  # Your code here


if __name__ == '__main__':
  tables_length = int(input())
  tables = []
  for _ in range(tables_length):
    tables.append(input())

  function_call_result = poker_table_traversal(tables)
  print(function_call_result)

File: Eclipse 2/codeLM starter-code/Java/PokerTableTraversal.java
import java.util.Scanner;

class PokerTableTraversal {
    static int pokerTableTraversal(String[] tables) {
        // Your code here
    }

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int tablesLength = Integer.parseInt(s.nextLine());
        String[] tables = new String[tablesLength];
        for (int i = 0; i < tablesLength; i++) {
            tables[i] = s.nextLine();
        }

        int functionCallResult = pokerTableTraversal(tables);
        System.out.println(functionCallResult);
    }
}